library/geolocation/src/main/java/com/epmedu/animeal/geolocation/location/Ext.kt

```diff
@@ -0,0 +1,12 @@
+package com.epmedu.animeal.geolocation.location
+
+import com.mapbox.geojson.Point
+import com.mapbox.turf.TurfMeasurement
+
+private const val NEAREST_DISTANCE_KM = 10
```

Comment:
Rename to default_nearest_distance_km, smth simillar
Not sure about private modifier

Reply:
Renamed, not needed at any other place.